app/src/main/java/fr/paug/androidmakers/ui/adapter/ScheduleDayAdapter.java

```diff
@@ -312,42 +314,53 @@ void bind(@NonNull final ScheduleSession scheduleSession, DaySchedule daySchedul
-                            sessionDuration, roomTitle, EmojiCompat.get().process(scheduleSession.getLanguageInEmoji())
-                    ));
+                    descriptionBuilder.append(resources.getString(R.string.session_description_placeholder_with_language,
+                            sessionDuration, roomTitle, EmojiCompat.get().process(scheduleSession.getLanguageInEmoji())));
                 }
             } else {
                 final int languageStringRes = Session.getLanguageFullName(scheduleSession.getLanguage());
-                final Resources resources = itemView.getResources();
                 if (roomTitle.isEmpty()) {
                     if (languageStringRes != 0) {
-                        sessionDescription.setText(resources.getString(R.string.session_description_placeholder,
-                                sessionDuration, resources.getString(languageStringRes)
-                        ));
+                        descriptionBuilder.append(resources.getString(R.string.session_description_placeholder,
+                                sessionDuration, resources.getString(languageStringRes)));
                     } else {
-                        sessionDescription.setText(sessionDuration);
+                        descriptionBuilder.append(sessionDuration);
                     }
                 } else {
                     if (languageStringRes != 0) {
-                        sessionDescription.setText(resources.getString(R.string.session_description_placeholder_with_language,
-                                sessionDuration, roomTitle, resources.getString(languageStringRes)
-                        ));
+                        descriptionBuilder.append(resources.getString(R.string.session_description_placeholder_with_language,
+                                sessionDuration, roomTitle, resources.getString(languageStringRes)));
                     } else {
-                        sessionDescription.setText(resources.getString(R.string.session_description_placeholder,
-                                sessionDuration, roomTitle
-                        ));
+                        descriptionBuilder.append(resources.getString(R.string.session_description_placeholder,
+                                sessionDuration, roomTitle));
+                    }
+                }
+            }
+
+            Session session = AgendaRepository.getInstance().getSession(scheduleSession.getSessionId());
+            if (session != null && session.speakers != null) {
+                for (int i = 0; i < session.speakers.length; i++) {
+                    final Speaker speaker = AgendaRepository.getInstance().getSpeaker(session.speakers[i]);
+
+                    if (speaker != null) {
+                        descriptionBuilder.append('\n');
+                        descriptionBuilder.append(speaker.getFullNameAndCompany());
```

Comment:
I think we don't need the company name

Reply:
I like information dense screens but fine with me